spec/support/analyzer/04_count_vs_size.rb

```diff
@@ -0,0 +1,21 @@
+ARRAY = (1..100).to_a
+
+def slow
+  ARRAY.count # return 100
+end
```

Comment:
The analyzer reports 3 violations, could you point to me which ones it is reporting on? Cause as much as I understand it is reporting on the ActiveRecord ones and we don't want that.

Reply:
I didn't check the receiver. Initially I thought it's a good idea to cover `ActiveRecord::Relation#count` as well, because its `#size` counterpart calls `#count` when relation has not been loaded. But it will prevent developer deliberately use `#count` when he/she wants to get "fresh" relation count